src/worker/users.js

```diff
@@ -95,7 +95,7 @@ class Users {
 
     async addUser(username, password) {
         if (!Helpers.validUsername(username)) {
-            return null;
+            throw new Error('Invalid username');
```

Comment:
Just curious, mind explaining where this change comes into place?

Reply:
If you tried to add a username that failed the validUsername test (in my case, 'x') it would say it succeeded and the user was added but nothing would go into the database. The code calling the addUser function was relying on exceptions to catch the DB errors and never actually checked if the function returned a null or not.